### DISCUSSION TITLE
最近威力加强了，纯自用搭建的也被墙了

### DISCUSSION THREAD

**Discussion by BitBitKing** [v2fly/v2ray-core#2097] [General]
技术栈：V2ray: vless+tcp+xtls
莫名无法使用，经监测：ICMP国内外无问题，TCP国外没问题，国内无法访问，定位端口被封。
此次域名/ip未被封锁，仅端口被封，端口并不是默认443，而是形如45678这样的非主流端口也未能幸免于难。
另一个搭建的ws+tls+sni然后用cf中转的目前没出现问题，待继续观测。
解决方案换一个端口即可，希望不会再被封，但只要出现了，猜测只是时间问题，目前不清楚具体封锁原因和手段，有同病相怜的可以一起分析下情况。
PS：未发布任何违规内容及言论，只是正常上网使用，估计有时候youtube自动跳到了一些视频，视频关键词敏感？只是作为猜测。只能说技术很强大。
<img width="765" alt="tcp" src="https://user-images.githubusercontent.com/111197555/197509771-79fac0d9-5dbc-419b-82f4-1f2791246e82.png">

**Comment by qiangliu8183** [comment]
我这里一样，3个小鸡都挂了，vmess+tls+ws，用了好几年了都没事，开会前陆续被封443端口

**Reply by BitBitKing** [reply]
换了端口又封了，什么也没做睡醒就被封了，还不是443端口，这力度有点强吧，无解了吗？

**Reply by ethenlong** [reply]
我最近是一天换一个ip，好在lightsail换ip不要钱，但这也不是长久之计，有大佬有其他推荐的梯子吗？

**Reply by wanghuibin0** [reply]
我也是，最近一段换了好几个，太折腾了

**Reply by BitBitKing** [reply]
命途多舛啊，heroku也不免费了，这边还一直封端口，之前常年用都没问题的

**Reply by hsg666888** [reply]
> 我最近是一天换一个ip，好在lightsail换ip不要钱，但这也不是长久之计，有大佬有其他推荐的梯子吗？

在边境拉一条短网线使用国外的网络上网，源生越南IP，物理外挂就是牛，他先封都封不了

**Reply by SPRCSY** [reply]
> > 我最近是一天换一个ip，好在lightsail换ip不要钱，但这也不是长久之计，有大佬有其他推荐的梯子吗？
> 
> 在边境拉一条短网线使用国外的网络上网，源生越南IP，物理外挂就是牛，他先封都封不了

这个真就私设信道了，辩解都辩解不了的，因为这个符合信道的定义[doge]

**Reply by hsg666888** [reply]
确实是，但是在边境两边商务合作需要，是直接来到营业厅那里托管，但是营业厅不会去监控你，而且那些营业厅都是外包的，可以放心使用，有点小贵

**Comment by kulongwangzhi85** [comment]
lightsail 几个月了，20 正常使用。目前正常。

**Comment by Eijux** [comment]
IP没问题，但是所有端口都没了……

**Reply by BitBitKing** [reply]
是这样的，端口换了，什么也没干，睡一觉起来又没了，真是强大啊

**Reply by wanghuibin0** [reply]
最近恶心死了

**Reply by xiaorong61** [reply]
试试 udp 协议的 TUIC 翻墙

**Comment by loophe** [comment]
不要换IP，有条件的话就在服务器上附加多一个IP，目前附加IP没有问题。

**Reply by vicacheung** [reply]
这是什么原理？

**Reply by loophe** [reply]
更新一下，附加ip也被墙了：（
换了浮动ip的服务器，开机一次变一次ip

**Reply by KEVIN45424** [reply]
我套cf也被封了

**Reply by itsalongstory** [reply]
> 我套cf也被封了

套 cf 是怎么封的，封域名？

**Reply by swatchion** [reply]
TCP RST 封的

**Reply by vicacheung** [reply]
> 更新一下，附加ip也被墙了：（ 换了浮动ip的服务器，开机一次变一次ip

还有这种机器嘛？大佬推荐一下

**Comment by vicacheung** [comment]
你们还看看网站啥的，我就连上去玩玩ingress也被封了。
封锁步骤就是喜闻乐见的首先封443，如果换端口IP全封。换IP用SS的话，直接全封。

**Reply by vicacheung** [reply]
前段时间封的443解封了，啥情况？

**Comment by wanghuibin0** [comment]
是啊，太烦人了，现在有什么比较稳的方法吗？

**Reply by BitBitKing** [reply]
好像除了换并祈祷别再被发现，别无他法

**Reply by xiaorong61** [reply]
TUIC

**Reply by iopq** [reply]
Yes, reality or hysteria (port hopping recommended though)

**Comment by loophe** [comment]
看到上面大家说几乎什么都没干就封ip，那也不能怪v2ray的vmess协议不够保密了，因为可以说，只要监测到有流量就封端口，或者封ip

**Reply by KEVIN45424** [reply]
我vless没被封，但wmess被封了。。。

**Comment by Kieap** [comment]
我的甲骨文白嫖服务器，443端口也被墙了，其他端口暂时正常

**Comment by DolorHunter** [comment]
bwh服务器，wss 配置443有web，封完就随便开一个端口，没用三天也封了。wss+web一套配置冲了三年什么事情没有，不应该啊。

**Comment by huguojunsy** [comment]
azure服务器，vmess协议，最近4个多月，差不多一天换一个ip。。。。心累。

**Reply by DolorHunter** [reply]
先套个cdn保命一下吧，真的感觉玩不过gfw了。线路是gia，套完感觉速度也没掉多少，cf就是我大哥。

**Reply by tixo** [reply]
我套cf 直接速度拦腰斩，哎心累

**Reply by laaArmy** [reply]
还用vmess？

**Reply by MakeFortuneQuietly** [reply]
商业机场都还在用vmess，墙早就能被识别了，只不过没收网。要是出问题了肯定就跑路+换马甲。
新协议不普及的一个主要原因是win版的v2rayn订阅只能识别vmess，ss。机场主不能指望小白用户自己导配置吧？

**Reply by otokaze** [reply]
vless+tls也封 问题在tls

**Reply by laaArmy** [reply]
> 商业机场都还在用vmess，墙早就能被识别了，只不过没收网。要是出问题了肯定就跑路+换马甲。
> 新协议不普及的一个主要原因是win版的v2rayn订阅只能识别vmess，ss。机场主不能指望小白用户自己导配置吧？

因为机场带中转

**Reply by sendguard** [reply]
> 商业机场都还在用vmess，墙早就能被识别了，只不过没收网。要是出问题了肯定就跑路+换马甲。 新协议不普及的一个主要原因是win版的v2rayn订阅只能识别vmess，ss。机场主不能指望小白用户自己导配置吧？

新协议可以尝试使用mihomo的客户端，等v2rayN支持新协议再换回来也不迟，目前v2rayN已经支持Hysteria协议了。

**Comment by winfet** [comment]
我的也是同样情况, 有没有什么解决办法, 我因为是自用的流量少, 换一个端口能坚持几天

**Comment by KEVIN45424** [comment]
代理下速度还可以8k能跑只是比较卡



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年1月6日(星期五) 晚上9:10
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [v2fly/v2ray-core] 最近威力加强了，纯自用搭建的也被墙了 (Discussion #2097)





 
我套cf 直接速度拦腰斩，哎心累

**Comment by BitBitKing** [comment]
现在封的也太频繁了，只能换端口，有时候换完坚持不到一天，什么也没干，睡一觉醒来就不能用，换一段端口后就该换ip了，真是无穷无尽啊，大家推荐什么技术栈啊？

**Reply by vicacheung** [reply]
纯技术不灵了，估计得上线路😂

**Reply by BitBitKing** [reply]
感觉现在啥线路都没用啊，道高一尺，墙高一丈啊

**Comment by siben168** [comment]
+1, 我用了近1年tls都没问题, ip基本上不会被封了. 但是最近几个月封端口太厉害了, 基本上一天一封. 大家有什么好办法分享一下.

**Reply by BitBitKing** [reply]
IP很少被封，基本都是端口封，xtls也不行

**Comment by forrestgao** [comment]
同样啊，2023年感觉端口挂的越来越频繁了，之前都用的好好的

**Reply by lijunjiang1989** [reply]
2023年好频繁，之前很少被封，我用的Azure的，现在基本每周都要换一次ip，而且是所有端口都封了，ssh都连不上。

**Reply by MakeFortuneQuietly** [reply]
封了就换协议啊，换ip有什么用。三番五次被断网还再犯，不怕被收网么

**Reply by hy3225569** [reply]
> 2023年好频繁，之前很少被封，我用的Azure的，现在基本每周都要换一次ip，而且是所有端口都封了，ssh都连不上。

和我一样，一直在换端口，端口用完了就换ip，不知道啥原因，难道因为域名的原因？

**Reply by kingwilliam** [reply]
有一些小细节可以研究一下

1. 不要直接 ssh vps. (那即是对墙说 这个vps是你维护的 请多多关照一下)
2. 尽可能不要用 tls+ws 和 tls+tcp, 可试用tls+h2/grpc, 但只可能会好一点点. (就是关於早前说的 tls over tls)

**Reply by tomoyo** [reply]
> > 2023年好频繁，之前很少被封，我用的Azure的，现在基本每周都要换一次ip，而且是所有端口都封了，ssh都连不上。
> 
> 和我一样，一直在换端口，端口用完了就换ip，不知道啥原因，难道因为域名的原因？

猜测和域名有些关系，我这边ss用ip直接访问一直用着没什么问题，最近需要给别人用就加强了一下防御，弄了个域名然后前端加上nginx伪装成站，结果睡一觉起来443就没了，是不是对域名有什么特殊照顾？

**Reply by tomoyo** [reply]
> 有一些小细节可以研究一下
> 
> 1. 不要直接 ssh vps. (那即是对墙说 这个vps是你维护的 请多多关照一下)
> 2. 尽可能不要用 tls+ws 和 tls+tcp, 可试用tls+h2/grpc, 但只可能会好一点点. (就是关於早前说的 tls over tls)

感觉没啥用？前端80没有封，443就封了，而且日志里面也没有看到有什么特殊访问，感觉现在的规则是“有流量但是不让我看？我先封了你再说”

**Reply by birdeagles** [reply]
我覺得就是這個原因，它是寧殺錯不放過，自建站是吧？不備案是吧？從國内訪問是吧？通通給你封了！

**Comment by jht3QAQ** [comment]
今天我这xtls-rprx-vision的机子也被封了 表现为RST攻击 端口是非常规端口 感觉xtls-rprx-vision也不安全了啊

**Reply by MakeFortuneQuietly** [reply]
这不是xray的协议么，怎么跑v2fly来讨论了

**Comment by mbicc** [comment]
我用的SS+KCPTUN.升级了端口段以后它封不了了...直接监听1000个端口+
但是我刷电报有时候没流量...感觉QOS...回头来看这个嘛一个端口真不行.我白鸡的TCP早封了

**Comment by timwuhaotian** [comment]
vultr v2ray 日本ip 用了两天就直接ip封了 不管拿什么域名来挂
![image](https://github.com/v2fly/v2ray-core/assets/1982947/de5f8932-978d-4f81-83f6-ac958bc1f010)

**Reply by BitBitKing** [reply]
封锁技术强大啊，我最近没被封，还挺好的，可能之前那个时间段比较敏感威力太猛

**Reply by ccfish86** [reply]
vultr不敢用了，当年用SS被直接封IP后，就换别的了。
现在是racknerd，巨慢，挂了WSS （已封），80端口还在，放了一个给儿科医生整理数据的工具*（WEB站）。其他端口还在，用得提心吊胆的。
我也不干啥啊，就是上上Github交个朋友，有时候google一下技术问题。

**Comment by yizhishang** [comment]
我月初买的vps，装上ss，用了没几天就封ip了。
前几天发现ip能通，于是装上了v2ray，加了的Trojan-WS-TLS和VLESS-VWS-TLS。然后今天发现80端口和443端口就访问不了了，GFW真的强！

**Reply by yizhishang** [reply]
> 我月初买的vps，装上ss，用了没几天就封ip了。 前几天发现ip能通，于是装上了v2ray，加了的Trojan-WS-TLS和VLESS-VWS-TLS。然后今天发现80端口和443端口就访问不了了，GFW真的强！

套了一层cloudflare都没有用

**Reply by hy3225569** [reply]
有用啊  就是速度慢了  https://github.com/233boy/v2ray/wiki/%E4%BD%BF%E7%94%A8Cloudflare%E4%B8%AD%E8%BD%ACV2Ray%E6%B5%81%E9%87%8F

**Reply by yizhishang** [reply]
我持续使用基本不超过一天，就要被墙

**Reply by cjjjjjj** [reply]
好像最近一直有加强, 挺要命的，有没有大佬有好的办法

**Comment by Qilely** [comment]
我也是动不动就封端口，不过60000多个慢慢封，一天一个还要封20年。唯一难受的问题是没了443和80，上自建的网站还要加端口号T_T

**Reply by qianbinbin** [reply]
端口封多了就该封 IP 了，以前我用 shadowsocks 就是这样的，最好还是套个 CDN 把真实 IP 藏好

**Comment by yizhishang** [comment]
优选IP+CDN

**Reply by BitBitKing** [reply]
套cf太慢，而且只支持443等几个端口
好多人遇到优选ip被gfw针对的情况
https://github.com/XIU2/CloudflareSpeedTest/issues/217

**Reply by kulongwangzhi85** [reply]
试试shadowtls用着不错

**Comment by 114514ns** [comment]
不可能是YouTube的原因
墙怎么可能看到你流量内容

**Reply by linuxbelletian** [reply]
协议有特征 被查水表了

**Reply by cjjjjjj** [reply]
你这是哪里，被你说的怪害怕的，是不是vps服务商把你交出去了？

**Reply by sendguard** [reply]
一般通过GFW的流量都是加密的，只有成功到达节点服务器才会被解密。
莫非GFW会主动解密流量？

**Reply by 114514ns** [reply]
> 一般通过GFW的流量都是加密的，只有成功到达节点服务器才会被解密。 莫非GFW会主动解密流量？

怎么可能 tls怎么可能被破解

**Comment by RockChinQ** [comment]
用 https://guide.v2fly.org/advanced/wss_and_web.html 所述的方案，caddy+v2ray搭到443，没做伪装网页，第二天443端口就被封了，请问使用伪装网页可以缓解吗？或者改成trojan或者尝试gRPC有效吗？

**Reply by MakeFortuneQuietly** [reply]
三件套已经是最高级的防御了，tls卡了你ip改什么都没有用。套cdn吧

**Comment by hcws** [comment]
请问使用国内的VPS+nginx 代理到国外行么

**Comment by hcws** [comment]
请问使用国内的VPS+nginx 代理到国外行么

**Comment by FreeSlaver** [comment]
真的没必要，太折腾了，用机场，便宜稳定，撒事不用操心，[bywave](https://user.by.ltd/aff.php?aff=17672)，
这个机场我18年末用到现在，当初有个套餐是250gb流量包终身100块还是80块，我充了100，现在账户还有35余额了，
就Google下，也不看油管，奈飞。现在没这套餐了，最便宜的30块，速度杠杠的，流量一个人肯定够用。

**Reply by FreeSlaver** [reply]
。。嗯，现在流量还剩110g。。。

**Reply by sendguard** [reply]
机场会不会泄露用户数据，会不会定期清空连接日志就很难说了。

**Reply by FreeSlaver** [reply]
> 机场会不会泄露用户数据，会不会定期清空连接日志就很难说了。

这个我就不知道了，反正我没干撒坏事，也不怕泄露撒，也不浏览zz类反动的站点，偶尔看点成人东西应该问题不大把？
这个机场是真的稳定，这么多年应该是3到5次故障把，每次故障之后它都会补偿的。

**Comment by eric-gitta-moore** [comment]
自建机场感觉太糟心了，管这里管那里的。我自建的就一个vps挂google one vpn，用其他机场做前置，主要是改善速度，机子没有被墙就是线路垃圾

**Reply by BitBitKing** [reply]
最近可能开会吧，自建的又频繁被封，我现在用这个了 [cloud](https://tinyurl.com/wgetcloud)

**Comment by zzabolazza** [comment]
搭完刚用一天就被封了。

**Reply by 2h0000** [reply]
我他喵搭完1s就被封了，就跑了40kb流量，套上cf也没用，再过几天不行就付费换ip了
![test40](https://github.com/v2fly/v2ray-core/assets/125747262/1f10115d-8f69-4f2d-8739-b7cd2707f89b)

**Comment by BitBitKing** [comment]
我太经常换ip了，麻木了

**Reply by yuzhoua** [reply]
我第一个节点用了快一个月，被封后换个新端口号，睡一觉就封了，有啥解决办法吗，只能换？

**Reply by 114514ns** [reply]
自建的用了一年啥事没有  前几天封了 过了几天又给我解开来了

**Reply by yuzhoua** [reply]
还真是，我都准备放弃了，前一段时间一链接，发现又给我放开了

**Comment by BitBitKing** [comment]
换一下ip，最好在换一下技术栈组合试试

**Reply by FengZi-lv** [reply]
楼主现在什么技术栈？
http/2+tls+web怎么样？

**Comment by test-maxwell** [comment]
我测试一个多月了，直接用vmess+tls+ws就是容易被封端口，我从443一路换到435了，每封一次就换一次端口，最后套了一层Cloudflare中转，已经20多天没被封了。你们可以试试，我用的是这个一键cf的脚本。
https://p3terx.com/archives/cloudflare-warp-configuration-script.html/comment-page-1

**Reply by ccfish86** [reply]
vmess+tls+ws 就是容易被封，我的被封半年多才解开。
同服务器 不加ws（端口1443）就一直没有被封，当然我只是看看github，不乱搜乱看其他东西。

**Reply by yuzhoua** [reply]
github还好，我这不翻也能看

**Comment by shazig** [comment]
完蛋了被，一觉醒来第一件事就是更滑端口，最近个把月就是这样

**Reply by yuzhoua** [reply]
我现在有个问题，我是用的甲骨文vps，现在国内访问过去很慢，导致我部署在vps上的frp完全不能用了，不知道是不是和这一波有关系，直接将国外的ip限速了

**Comment by xiaohecloud** [comment]
我的也是，一觉醒来就封端口好几次了，不过我v6的机从来没被封过

**Reply by xiaohecloud** [reply]
你可以用v6的机看看

**Comment by panluoyant** [comment]
我建议是使用VMESS+ws+tls+CDN目前感觉良好

**Reply by luogeshibu** [reply]
我的已经不行了

**Comment by hsg666888** [comment]
ipv6我还没有试过，我试试看，如何能二级代理的话应该更好的防封

AMDyes97 ***@***.***> 于 2024年10月5日周六 下午6:27写道：

**Comment by hsg666888** [comment]
我试试看

panluoyant ***@***.***> 于 2024年10月5日周六 下午6:30写道：

**Comment by luogeshibu** [comment]
vmess+ws+tls+nginx+cdn我这个被封了，各位有没有被封的？

**Comment by MangoHobo** [comment]
shaowsocks 4.4.1.0 + v2ray-plugin + nginx 
每天换一个端口即可解决，无关流量大小（已验证）。
总体解决方法感觉可以使用域名指向该ip 每次通过服务器防火墙转发内部端口至该端口，写个定时任务启动docker服务 更多端口或者一口气启动一个月或更久得端口量但不开放防火墙即可
aliyun_jp_弹性ipv4

**Reply by FreeSlaver** [reply]
没撒用，等开关键zz会议，照封不误

**Reply by banbadaowd83** [reply]
Aliyun JP 最特么容易被封端口了，最近上头焦躁中。

**Reply by MangoHobo** [reply]
好麻烦 ，不知道有解决方案踢我一脚。

**Reply by iopq** [reply]
hysteria allows you to automatically change ports every x seconds

**Reply by hsg666888** [reply]
我的套二级代理好几个月没封过，好好的

**Reply by banbadaowd83** [reply]
换了reality ，好很多了。。

**Reply by momaek** [reply]
用上 reality 2天 IP 都没了 ... bwh BIGGER BOX 买了一周 然后就走 Refund 了。。。

**Reply by MangoHobo** [reply]
> 用上现实2天IP都没有了...bwh BIGGER BOX买了一周然后就走退款了。。。

一般就是写个脚本每天自动换端口我目前是按日期自动更换端口 20250409，端口就是50409大概这样每天换一下也还可以接受目前没被封了

**Reply by MangoHobo** [reply]
> hysteria 允许你每 x 秒自动更改端口

I achieved it through Linux sh+scheduled tasks, automated scripts, and daily port switching based on dates

**Comment by yue-fei** [comment]
最近每天被封一个端口，CDN 换了域名也没用。

**Comment by yjianzhu** [comment]
22端口都给我干没了，感觉是封了我ip阿

**Reply by MangoHobo** [reply]
我的也没了，换个21试试 大概率是上行比下载高。

**Reply by MangoHobo** [reply]
忘记补充了，22没了 ip还正常

**Comment by xiaorong61** [comment]
Shadowsocks 2022

**Comment by adventure515** [comment]
我的节点套了Gcore 的CDN无法连接，测速-1是不是被强了，但是XUI建的vmess+ws节点还能用，去Gcore官网看cdn也还是活动状态😂

**Comment by KKCAT-77** [comment]
我换了五家机场都没用，上传速度最高只有200-300KB/s，下载能跑满300兆；国内测速下载游戏网盘上传一切正常(下载300兆，上行50-60兆)。。。就tm的挂梯子后上传拉跨！直连的，什么IEPL的线路都试过了，都没用。。。求解！😭 (补充一点：之前挂梯子用BT下了几十个G片子，我忘记了不能用梯子下bt的，为了排除这个因素所以换了很多家机场；后面挂梯子上传的网速变慢之后，上门的电信师傅说我的红米ac2100路由器被标记为跑pcdn的设备了！然后他帮我换个了中兴的路由器，挂梯子上传速度还是200-300kb/s)

**Reply by MangoHobo** [reply]
被标记可以投诉，推荐试试蜂窝机场，只是建议...我用的都是自建的，偶尔用用蜂窝
